source/sdk/threading/Thread.ooc

```diff
@@ -17,6 +17,8 @@ Thread: abstract class {
 
 	wait: abstract func ~timed (seconds: Double) -> Bool
 
+	detach: abstract func -> Bool
```

Comment:
It's like that in many places where there are only abstract functions. I will look into it for another PR.